apps/server/migrations/0011_measurement.ts

```diff
@@ -18,6 +18,12 @@ export async function up(db: Kysely<unknown>): Promise<void> {
     .addColumn("test_id", "text", (col) =>
       col.notNull().references("test.id").onDelete("cascade"),
     )
+    .addColumn("session_id", "text", (col) =>
+      col.notNull().references("session.id").onDelete("cascade"),
+    )
```

Comment:
A session can be run in cycle, so we could add a (optional) `sequence_name`. That we way would be able to group them in the UI using this and the timestamp ?

Reply:
or maybe we attach a cycle number with each measurement?

Reply:
Smart